label-inspector/label_inspector.ipynb

```diff
@@ -796,75 +794,861 @@
+       "      <td>0.039741</td>\n",
+       "      <td>0.026302</td>\n",
+       "      <td>0.029070</td>\n",
+       "      <td>0.835001</td>\n",
+       "      <td>0.069886</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  letter_grade  A_probability  B_probability  C_probability  D_probability  \\\n",
+       "0            B       0.074430       0.651123       0.032060       0.076554   \n",
+       "1            A       0.937403       0.002210       0.002298       0.000344   \n",
+       "2            C       0.060622       0.075482       0.740313       0.101991   \n",
+       "3            F       0.129802       0.008672       0.002780       0.025871   \n",
+       "4            D       0.039741       0.026302       0.029070       0.835001   \n",
+       "\n",
+       "   F_probability  \n",
+       "0       0.165833  \n",
+       "1       0.057744  \n",
+       "2       0.021591  \n",
+       "3       0.832874  \n",
+       "4       0.069886  "
+      ]
+     },
+     "execution_count": 36,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "real_time_preds = pd.read_csv(real_time_output_file)\n",
+    "real_time_preds.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Similar to above, the output contains two pieces of information:\n",
+    "\n",
+    "- the first column contains the predicted label for each example\n",
+    "- the remaining columns contain the predicted class probabilities for each example, these columns will have the names `[class_label]__probability`\n",
+    "    - in our sample dataset, the `A__probability` column shows the model predicted probability that an example should have the letter grade A"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 7. Clean Up Resources "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Delete models endpoints (skip if you skipped Section 6)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you are done with performing real-time inferences, you no longer need the saved model and deployed endpoint. Here we show how to terminate the endpoint to avoid being charged."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# predictor.delete_model()\n",
+    "# predictor.delete_endpoint(delete_endpoint_config=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also delete the model associated with the batch transform job."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# transformer.delete_model()"
+   ]
+  },
+  {
```

Comment:
maybe include a link at the very bottom user can click to see their Sagemaker deployments in aws console in order to verify themselves these endpoints are deleted?

Reply:
text next to the link can say user can manually delete the endpoints from there too